$OpenBSD: patch-plugins_project-wizard_druid_c,v 1.1.1.1 2011/07/12 11:28:50 jasper Exp $

OpenBSD doesn't have PackageKit support, so don't bother showing
the button to install missing packages.

--- plugins/project-wizard/druid.c.orig	Tue Jul 12 10:53:59 2011
+++ plugins/project-wizard/druid.c	Tue Jul 12 10:54:47 2011
@@ -916,12 +916,14 @@ check_and_warn_missing (NPWDruid *druid)
 		hbox = gtk_hbox_new (FALSE, 0);
 		gtk_widget_show (hbox);
 
+#ifndef __OpenBSD__ /* Lacks PackageKit support */
 		install_button =
 			gtk_button_new_with_label (_("Install missing packages"));
 		gtk_box_pack_end (GTK_BOX (hbox), install_button, FALSE, FALSE, 10);
 		g_signal_connect (install_button, "clicked",
 						  G_CALLBACK (on_install_button_clicked), druid);
 		gtk_widget_show (install_button);
+#endif
 
 		npw_druid_fill_error_page (druid, hbox,
 								   GTK_MESSAGE_WARNING,
